core/libdeno.rs

```diff
@@ -750,6 +145,7 @@ impl AsRef<[u8]> for deno_buf {
 /// but the existence of a PinnedBuf inhibits this until it is dropped. It
 /// behaves much like an Arc<[u8]>, although a PinnedBuf currently can't be
 /// cloned.
+#[allow(unused)]
 pub struct PinnedBuf {
```

Comment:
Is it unused?

Reply:
`backing_store` field is unused, I'll move attribute